modules/core/server/views/partials/footer.server.view.html

```diff
@@ -5,25 +5,39 @@
   <hr>
   <div class="navbar">
     <div class="col-xs-6">
-      <ul class="navbar">
+      <ul class="list-unstyled">
+        <li><a ui-sref="volunteering">Volunteering</a></li>
+        <li><a href="https://meta.trustroots.org/t/lets-translate-trustroots/59">Translations</a></li>
+        <li><a ui-sref="rules">Rules</a></li>
+        <li><a ui-sref="privacy">Privacy</a></li>
+        <li><a ui-sref="donate">Donate</a></li>
+        <li><a ui-sref="faq">FAQ</a></li>
         <li itemscope itemtype="http://schema.org/Organization">
           <a ui-sref="foundation" itemprop="name">
             Trustroots Foundation
           </a>
-          <link itemprop="url" href="https://www.trustroots.org/">
-          <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
+          <link itemprop="url" href="{{hostPort}}">
+          <link itemprop="logo" href="{{hostPort}}/img/logo/color.png">
         </li>
-        <li><a ui-sref="privacy">Privacy</a></li>
         <li><a ui-sref="contact">Contact</a></li>
-        <li><a ui-sref="faq">FAQ</a></li>
-        <li><a ui-sref="volunteering">Volunteering</a></li>
-        <li><a ui-sref="rules">Rules</a></li>
-        <li><a ui-sref="rules">Rules</a></li>
-        <li><a ui-sref="donate">Donate</a></li>
       </ul>
     </div>
-    <div class="col-xs-6 pull-right">
+    <div class="col-xs-6">
       <ul class="list-inline">
+        <li>
+          {% if appSettings.commit %}
+          {% if appSettings.commitTime %}Last change {{ appSettings.commitTime }}{% endif %}
```

Comment:
Honestly to me this is still just noise and I think there are better ways to attract volunteers, but if you really want it there... :-D

Reply:
Discussion:
https://meta.trustroots.org/t/show-last-date-the-code-was-changed/87